spec/content_metadata_spec.rb

```diff
@@ -616,30 +616,56 @@
       end
     end
 
-    context 'when using a 3d object with one 3d type files and three other supporting files (where one supporting file is a non-viewable but downloadable 3d file)' do
+    context 'when using a 3d object with one 3d type files and three other supporting files (where two supporting files are non-viewable but downloadable 3d file)' do
```

Comment:
should the last word of the sentence be changed to "files"?

Reply:
fixed